components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -74,6 +75,12 @@
         isActive: false,
         limitTo: 9
       };
+    },
+
+    methods: {
+      selected(value) {
+        this.$emit('selectedOptions', { [this.name]: [ ...value ] });
```

Comment:
This isn't so easy to scan.

Let's rewrite it as:
```js
        this.$emit('selectedOptions', this.name, value);
```
... with the accompanying change to `updateSelected`, above.